index.bs

```diff
@@ -49,29 +49,45 @@ conversion domain.
 
 # HTML monkeypatches # {#html-monkeypatches}
 
-Add the following content attributes to the <{a}> element:
+Add the following <a spec=html>content attributes</a> to the <{a}> element:
+
+: <{a/attributionsourceeventid}>
+:: Identifies the declared attribution source
+: <{a/attributiondestination}>
+:: Site which can attribute an event to the declared attribution source
+: <{a/attributionreportto}>
+:: [=url/origin=] to receive attribution reports
+: <{a/attributionexpiry}>
+:: Length of time the attribution souce is valid
+
+Extend the <{a}> element's <a spec=html>DOM interface</a> to include the following interface:
 
 <pre class="idl">
 partial interface HTMLAnchorElement {
-    [CEReactions, Reflect] attribute DOMString attributiondestination;
-    [CEReactions, Reflect] attribute DOMString attributionsourceeventid;
-    [CEReactions, Reflect] attribute DOMString attributionreportto;
-    [CEReactions, Reflect] attribute unsigned long long attributionexpiry;
+    [CEReactions] attribute DOMString attributiondestination;
```

Comment:
IDL attributes use camel case, so e.g. `attributionDestination`, `attributionSourceEventId`, `attributionReportTo`, `attributionExpiry`.

Reply:
Oops, thanks! For John, the rules are in https://w3ctag.github.io/design-principles/#casing-rules.